<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VesselSegmentationToolboxWidget</class>
 <widget class="QWidget" name="VesselSegmentationToolboxWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Image:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Vessel path:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="vesselTreeLineEdit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Vessel tree name</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Create or select a vessel tree</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="referenceImageLineEdit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Image name</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Select reference image</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Vessel tree:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="vesselPathLineEdit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Vessel path name</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Create or select a vessel path</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="createTreeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Create a new vessel tree</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources/vesselSegmentation.qrc">
         <normaloff>:/vesselSeg/icons/add.png</normaloff>:/vesselSeg/icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="createPathButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Create a new vessel path</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources/vesselSegmentation.qrc">
         <normaloff>:/vesselSeg/icons/add.png</normaloff>:/vesselSeg/icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QToolButton" name="duplicatePathButton">
       <property name="toolTip">
        <string>Duplicate current vessel path</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/vesselSegmentation.qrc">
         <normaloff>:/vesselSeg/icons/copy.png</normaloff>:/vesselSeg/icons/copy.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="createMaskedImageButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Create a new vessel tree</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources/vesselSegmentation.qrc">
         <normaloff>:/vesselSeg/icons/settings.png</normaloff>:/vesselSeg/icons/settings.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="editPathButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Edit selected vessel path&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Ctrl +&lt;/span&gt;&lt;img src=&quot;:/contourModeling/cursors/left-mouse.png&quot;/&gt; - add control point (&lt;img src=&quot;:/contourModeling/cursors/cursor_addpoint.png&quot; width=&quot;48&quot;/&gt;) or insert control point (&lt;img src=&quot;:/contourModeling/cursors/cursor_insertpoint.png&quot; width=&quot;48&quot;/&gt;) when hovering over the vessel path&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Shift + &lt;/span&gt;&lt;img src=&quot;:/contourModeling/cursors/left-mouse.png&quot;/&gt; - remove control point (&lt;img src=&quot;:/contourModeling/cursors/cursor_removepoint.png&quot; width=&quot;48&quot;/&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/vesselSegmentation.qrc">
         <normaloff>:/vesselSeg/icons/edit.png</normaloff>:/vesselSeg/icons/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="tensionSlider">
       <property name="toolTip">
        <string>Set the tension of the spline for the current vessel path</string>
       </property>
       <property name="minimum">
        <number>-99</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="addAutoButton">
          <property name="toolTip">
           <string>Add new control points to the closest end of the vessel path</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources/vesselSegmentation.qrc">
            <normaloff>:/vesselSeg/icons/addAuto.png</normaloff>:/vesselSeg/icons/addAuto.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="addToStartButton">
          <property name="toolTip">
           <string>Add new control points to the beginning of the vessel path</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources/vesselSegmentation.qrc">
            <normaloff>:/vesselSeg/icons/addToStart.png</normaloff>:/vesselSeg/icons/addToStart.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="addToEndButton">
          <property name="toolTip">
           <string>Add new control points to the end of the vessel path</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources/vesselSegmentation.qrc">
            <normaloff>:/vesselSeg/icons/addToEnd.png</normaloff>:/vesselSeg/icons/addToEnd.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="VesselPathTableViewWidget" name="vesselPathView">
     <property name="toolTip">
      <string>Control points</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VesselPathTableViewWidget</class>
   <extends>QTableView</extends>
   <header>src/internal/vesselPathTableViewWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>referenceImageLineEdit</tabstop>
  <tabstop>vesselTreeLineEdit</tabstop>
  <tabstop>createTreeButton</tabstop>
  <tabstop>vesselPathLineEdit</tabstop>
  <tabstop>createPathButton</tabstop>
  <tabstop>vesselPathView</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/vesselSegmentation.qrc"/>
 </resources>
 <connections/>
</ui>
